signature/async/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+//! RustCrypto: `async-signature` crate.
+//!
+//! This is an experimental crate containing `async` versions of select traits
+//! from the [`signature`] crate, namely [`AsyncSigner`] and when the `digest`
+//! feature is enabled, [`AsyncDigestSigner`].
```

Comment:
Note: this crate is presently only providing signing traits.

I didn't see the immediate use case for async signature verification, but if someone has one I can add `AsyncVerifier`/`AsyncDigestVerifier`.